.xcode-version

```diff
@@ -0,0 +1 @@
+~> 14.0
```

Comment:
The file we had before was `.xcversion`, but [the file used by Fastlane's `xcversion` action is supposed to be named `.xcode-version`](https://docs.fastlane.tools/actions/xcversion/) (by convention similar to files like `.ruby-version` and the like), not `.xcversion`, hence this file rename (in addition to updating its content to `~> 14.0`).